README.md

```diff
@@ -37,27 +59,16 @@ tuple in a list (`[{}]`) to represent empty objects. jsn is incompatible with
 this format. While the getter (`jsn:get/2,3`) functions are generally functional;
 most other library functions are not, and may result in unpredictable behaviors.
```

Comment:
Might want to mention that jsonx is basically abandoned. Choose jsone or something.